2022-01-09 15:51:38,252 INFO    MainThread:10990 [wandb_setup.py:_flush():71] setting env: {}
2022-01-09 15:51:38,252 INFO    MainThread:10990 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-09 15:51:38,252 INFO    MainThread:10990 [wandb_init.py:_log_setup():371] Logging user logs to /content/tbcnn/wandb/run-20220109_155138-3or247u5/logs/debug.log
2022-01-09 15:51:38,253 INFO    MainThread:10990 [wandb_init.py:_log_setup():372] Logging internal logs to /content/tbcnn/wandb/run-20220109_155138-3or247u5/logs/debug-internal.log
2022-01-09 15:51:38,253 INFO    MainThread:10990 [wandb_init.py:init():404] calling init triggers
2022-01-09 15:51:38,253 INFO    MainThread:10990 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-09 15:51:38,253 INFO    MainThread:10990 [wandb_init.py:init():460] starting backend
2022-01-09 15:51:38,253 INFO    MainThread:10990 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-09 15:51:38,269 INFO    MainThread:10990 [backend.py:ensure_launched():216] starting backend process...
2022-01-09 15:51:38,282 INFO    MainThread:10990 [backend.py:ensure_launched():222] started backend process with pid: 11001
2022-01-09 15:51:38,289 INFO    MainThread:10990 [wandb_init.py:init():469] backend started and connected
2022-01-09 15:51:38,305 INFO    MainThread:10990 [wandb_init.py:init():533] updated telemetry
2022-01-09 15:51:38,377 INFO    MainThread:10990 [wandb_init.py:init():563] communicating current version
2022-01-09 15:51:39,515 INFO    MainThread:10990 [wandb_init.py:init():568] got version response 
2022-01-09 15:51:39,516 INFO    MainThread:10990 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-09 15:51:39,740 INFO    MainThread:10990 [wandb_init.py:init():606] starting run threads in backend
2022-01-09 15:51:42,758 INFO    MainThread:10990 [wandb_run.py:_console_start():1810] atexit reg
2022-01-09 15:51:42,759 INFO    MainThread:10990 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-09 15:51:42,759 INFO    MainThread:10990 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-09 15:51:42,765 INFO    MainThread:10990 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-09 15:51:42,765 INFO    MainThread:10990 [wandb_init.py:init():633] run started, returning control to user process
2022-01-09 15:51:49,035 INFO    MainThread:10990 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-01-09 15:51:49,036 INFO    MainThread:10990 [wandb_run.py:_restore():1752] restore
2022-01-09 15:51:51,385 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 686
}

2022-01-09 15:51:51,654 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 686
}

2022-01-09 15:51:51,774 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 13592
}

2022-01-09 15:51:51,877 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 13592
}

2022-01-09 15:51:51,979 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 13592
}

2022-01-09 15:51:52,082 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}

2022-01-09 15:51:52,184 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}

2022-01-09 15:51:52,286 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}

2022-01-09 15:51:52,388 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}

2022-01-09 15:51:52,491 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}

2022-01-09 15:51:53,104 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}

2022-01-09 15:51:53,579 INFO    MainThread:10990 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13592
  total_bytes: 13592
}
local_info {
}

2022-01-09 15:51:55,288 INFO    MainThread:10990 [wandb_run.py:_append_files():2180] logging synced files
